### PR TITLE
ag: new package

### PR DESCRIPTION
**Pull request by moguriso**

### CODE REVIEW

File: ag/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Maintainer: Alexey Pavlov <alexpux@gmail.com>
+
+pkgname=ag
+pkgver=0.30.0.31.gdf5d456
+pkgrel=1
+pkgdesc='A terminal multiplexer'
```
Comment (Alexpux) [inline ag/PKGBUILD]: wrong.

File: ag/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+url='http://tmux.github.io/'
```
Comment (Alexpux) [inline ag/PKGBUILD]: wrong

File: ag/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+arch=('x86_64')
```
Comment (Alexpux) [inline ag/PKGBUILD]: wrong

File: ag/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+license=('Apache License 2.0')
+provides=('ag')
+conflicts=('ag')
+depends=('libpcre' 'mingw-w64-x86_64-pcre' 'mingw-w64-x86_64-xz' 'lzip' 'xz')
```
Comment (Alexpux) [inline ag/PKGBUILD]: wrong

**Comment by Alexpux** [comment]
Also we already have AG packages for mingw.

**Comment by moguriso** [comment]
I'm sorry. I didn't notice that.

**Pull request closed without merging**